README.md

```diff
@@ -59,14 +59,29 @@ cd gem5-aladdin && git pull origin master && git submodule update --init --recur
 cd LLVM-Tracer && git pull origin master && cd ..
 cd smaug && git pull origin master && git submodule update --init --recursive && cd ..
 ```
+# Building #
+We need to build gem5-Aladdin:
+
+```bash
+cd /workspace/gem5-aladdin
+python2.7 `whcih scons` build/X86/gem5.opt PROTOCOL=MESI_Two_Level_aladdin -j8
```

Comment:
Given that we're in Docker, we should be able to control the environment such that we don't need this ugly hack. SCons supports python3 now, is this a gem5 issue? I thought gem5 was fully python3 compatible now too.

Also, typo in "which scons".

Reply:
Yeah gem5 doesn't fully support Python3 yet - at least the current version we are based off of doesn't. But they now claim Python 3 is supported with Ubuntu 20.04, maybe time to do another big merge.

Reply:
Let's merge this one for now?

Reply:
Sure. It's super ugly, but let's fix it later...